docs/running-virtualenv.md

```diff
@@ -132,3 +132,23 @@ yarn run gulp test:unit       # Run only unit tests on source code.
 yarn run gulp test:acceptance # Run only acceptance (in-browser) tests on production code.
 yarn run gulp audit           # Run code quality audits.
 ```
+
+### Reinstalling the virtual environment
+
+To remove an existing virtual environment for 
+[a reinstall of cfgov-refresh](installation/#stand-alone-installation), 
```

Comment:
I think these links need to be made explicitly relative:

```suggestion
[a reinstall of cfgov-refresh](../installation/#stand-alone-installation), 
```

I notice there's at least one additional broken link above; the "the standalone installation instructions" link on [the live documentation](http://cfpb.github.io/cfgov-refresh/running-virtualenv/) is broken.

Reply:
Good catch! Will fix it.